eventWorker.py

```diff
@@ -24,12 +24,14 @@ def run(self):
             print "sleep for " + str(self.eventObject["seconds"]) + " seconds"
             time.sleep(int(self.eventObject["seconds"]))
 
-        if "command" in self.eventObject:
-            print "execute command: " + self.eventObject["command"]
-            exitCode = call(self.eventObject["command"], shell=True)
-            if exitCode != 0:
-                print "commend exited with exit code " + str(exitCode) + ". Abort."
-                return exitCode
+        if "commands" in self.eventObject:
```

Comment:
I would love the have backwards-compatibility. So it would great, if it also works with `command`.